inference/inference.go

```diff
@@ -310,7 +313,15 @@ func (s *Store) processQuad(q quad.Quad) {
 	case rdfs.Range:
 		s.setPropertyRange(sub, obj)
 	default:
-		s.addPropertyInstance(pred)
+		p := s.addPropertyInstance(pred)
+		domain := p.Domain()
+		if domain != nil {
+			domain.references++
+		}
+		_range := p.Range()
```

Comment:
Sorry, I was the one mentioning `_range`, but it would be better to rename it to `prange` (property range) or something similar (and short). Same in other places.